### PR TITLE
fix: new plans

### PR DESCRIPTION
**Pull request by t00f**
Revamp existing plans that were not exposed

> Project: https://github.com/orgs/aporeto-inc/projects/386

### CODE REVIEW

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "ssh_freemium",
    "component": "clad",
    "pr-id": "1056",
    "commit-sha": "3a87d8f74c6395ed3abbfcf2fc81350a3d80e4f7"
  },
  {
    "project": "ssh_freemium",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "efe4988c61e49711b942516c6afad622fba71356"
  },
  {
    "project": "ssh_freemium",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "b2e9d232de174888310b99719d7a95b2c8ff00bb"
  },
  {
    "project": "ssh_freemium",
    "component": "apoctl",
    "pr-id": "175",
    "commit-sha": "4c864b7f6d420d820b3bb32fa808e10fae93fc9d"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "ssh_freemium",
    "component": "apoctl",
    "pr-id": "175",
    "commit-sha": "4c864b7f6d420d820b3bb32fa808e10fae93fc9d"
  },
  {
    "project": "ssh_freemium",
    "component": "clad",
    "pr-id": "1056",
    "commit-sha": "3a87d8f74c6395ed3abbfcf2fc81350a3d80e4f7"
  },
  {
    "project": "ssh_freemium",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "efe4988c61e49711b942516c6afad622fba71356"
  },
  {
    "project": "ssh_freemium",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "b2e9d232de174888310b99719d7a95b2c8ff00bb"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "f9e55bb786a63b2686f9b553cd7ae55b5676d8b8"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "b2e9d232de174888310b99719d7a95b2c8ff00bb"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "ssh_freemium",
    "component": "clad",
    "pr-id": "1056",
    "commit-sha": "591359b3e70d4234b1554832d2b8bafbf2a23c45"
  },
  {
    "project": "ssh_freemium",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "8efbd85c47d235d4e10ec25bd489642b028c35b1"
  },
  {
    "project": "ssh_freemium",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "c85ce759a8bfd37066b93c0df48d080e1d8770f3"
  },
  {
    "project": "ssh_freemium",
    "component": "apoctl",
    "pr-id": "175",
    "commit-sha": "4c864b7f6d420d820b3bb32fa808e10fae93fc9d"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "01ae8d22e8ff8a526f9551d39ab2e5b0d3074a0e"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "6d52a3b74c5e0a7d320f12c12f666b2892cdd73a"
  },
  {
    "project": "",
    "component": "apoctl",
    "pr-id": "175",
    "commit-sha": "daad6a00ddc9f329acbcdedd6b26742c1ddcce67"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "6d52a3b74c5e0a7d320f12c12f666b2892cdd73a"
  },
  {
    "project": "",
    "component": "apoctl",
    "pr-id": "175",
    "commit-sha": "84e8ed5937a28b7d2b0d58f28f5578edfac1f2d3"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "7b309322b871dfaa922b0c5a4cbfa32d63c24f0c"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "ssh_freemium",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "c48173fb842f30ff6324f9a8687f9a0d36f93fb3"
  },
  {
    "project": "ssh_freemium",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "6d52a3b74c5e0a7d320f12c12f666b2892cdd73a"
  },
  {
    "project": "ssh_freemium",
    "component": "apoctl",
    "pr-id": "175",
    "commit-sha": "84e8ed5937a28b7d2b0d58f28f5578edfac1f2d3"
  },
  {
    "project": "ssh_freemium",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "ssh_freemium",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  },
  {
    "project": "ssh_freemium",
    "component": "clad",
    "pr-id": "1056",
    "commit-sha": "9991ee4a6d0b3993461804a400cc672e1d053680"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "c48173fb842f30ff6324f9a8687f9a0d36f93fb3"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "6d52a3b74c5e0a7d320f12c12f666b2892cdd73a"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "c48173fb842f30ff6324f9a8687f9a0d36f93fb3"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "6d52a3b74c5e0a7d320f12c12f666b2892cdd73a"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "c48173fb842f30ff6324f9a8687f9a0d36f93fb3"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "c45dc1147317e1086d5b1508ee4096e2364550dc"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "ssh_freemium",
    "component": "clad",
    "pr-id": "1056",
    "commit-sha": "1a507c9ca3f0bade179e1614f9f8167827df008f"
  },
  {
    "project": "ssh_freemium",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "c48173fb842f30ff6324f9a8687f9a0d36f93fb3"
  },
  {
    "project": "ssh_freemium",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "c45dc1147317e1086d5b1508ee4096e2364550dc"
  },
  {
    "project": "ssh_freemium",
    "component": "apoctl",
    "pr-id": "175",
    "commit-sha": "84e8ed5937a28b7d2b0d58f28f5578edfac1f2d3"
  },
  {
    "project": "ssh_freemium",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "ssh_freemium",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

File: specs/ldapprovider.spec [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  delete:
+    description: Deletes the object with the given ID.
+    global_parameters:
+    - $filtering
+  extends:
+  - '@base'
+  - '@described'
+  - '@identifiable-stored'
+  - '@timeable'
+  - '@named'
```
Comment (primalmotion) [inline specs/ldapprovider.spec]: missing zoneable

File: specs/oidcprovider.spec [hunk 2]
```diff
@@ -19,12 +19,8 @@ model:
...
   - '@identifiable-stored'
   - '@timeable'
-
-# Indexes
-indexes:
-- - :unique
-  - parentID
-  - name
+  - '@base'
+  - '@named'
```
Comment (primalmotion) [inline specs/oidcprovider.spec]: missing zonable

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "ssh_freemium",
    "component": "apoctl",
    "pr-id": "175",
    "commit-sha": "84e8ed5937a28b7d2b0d58f28f5578edfac1f2d3"
  },
  {
    "project": "ssh_freemium",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "dcda3597e0cfff4f584be00575c00097618c83f0"
  },
  {
    "project": "ssh_freemium",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "ssh_freemium",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  },
  {
    "project": "ssh_freemium",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "2cff4037fab9452260374c054f1ca2b22c1ece0f"
  },
  {
    "project": "ssh_freemium",
    "component": "clad",
    "pr-id": "1056",
    "commit-sha": "69dbd5809704458540585c33f2c662e70238adda"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "ssh_freemium",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "a6090509a8a6e8ee212aaa2ecbb5470dd2796e80"
  },
  {
    "project": "ssh_freemium",
    "component": "clad",
    "pr-id": "1056",
    "commit-sha": "69dbd5809704458540585c33f2c662e70238adda"
  },
  {
    "project": "ssh_freemium",
    "component": "apoctl",
    "pr-id": "175",
    "commit-sha": "84e8ed5937a28b7d2b0d58f28f5578edfac1f2d3"
  },
  {
    "project": "ssh_freemium",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "dcda3597e0cfff4f584be00575c00097618c83f0"
  },
  {
    "project": "ssh_freemium",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "ssh_freemium",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "1e4d5e01f6e0dd0d344cf05c91c1d90765808268"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "dcda3597e0cfff4f584be00575c00097618c83f0"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "55322b3fc27bccdee2c5445eea0bb4ed39d85e20"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "f94913c18eb3ed3bcc48c49d25cac91ae55e431d"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "feb98a17cdfeb80934886cb6df1693671ed0958f"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "f94913c18eb3ed3bcc48c49d25cac91ae55e431d"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "f90173f537cf56c64781511d169c1d220e8466f0"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "f94913c18eb3ed3bcc48c49d25cac91ae55e431d"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "f90173f537cf56c64781511d169c1d220e8466f0"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "f94913c18eb3ed3bcc48c49d25cac91ae55e431d"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "f90173f537cf56c64781511d169c1d220e8466f0"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "f94913c18eb3ed3bcc48c49d25cac91ae55e431d"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "f94913c18eb3ed3bcc48c49d25cac91ae55e431d"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "f90173f537cf56c64781511d169c1d220e8466f0"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "359",
    "commit-sha": "f90173f537cf56c64781511d169c1d220e8466f0"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "298",
    "commit-sha": "5a495f839a8d19d502b7d31f00256be9dff253a4"
  },
  {
    "project": "",
    "component": "midgard-lib",
    "pr-id": "20",
    "commit-sha": "ce5e27a9a9a37502a61945941cac76d7f1bc22a2"
  },
  {
    "project": "",
    "component": "underwater",
    "pr-id": "58",
    "commit-sha": "7b7860c4936a85b355dc5f74102cf2322db69567"
  }
]
```

**Pull request merged**